### PR TITLE
Docs: Burrow Build Note

### PR DESCRIPTION
**Pull request by LiraLemur**
Signed-off-by: Sara <sara@Sara.Sara>

https://jira.hyperledger.org/browse/IR-924

### Description of the Change

Building Iroha with Burrow and without GO doesn't work.
Added a note about the need to install GO. 

### Benefits

Better docs

### Possible Drawbacks 

None

### CODE REVIEW

File: docs/source/build/index.rst [hunk 1]
```diff
@@ -293,6 +293,8 @@ Main Parameters
 
 .. note:: If you would like to use HL Ursa cryptography for your build, please install `Rust <https://www.rust-lang.org/tools/install>`_ in addition to other dependencies. Learn more about HL Ursa integration `here <../integrations/index.html#hyperledger-ursa>`_.
 
+  If you want to use HL Burrow integration, do not forget to install `Go <https://golang.org/doc/install>`_. Learn more about `HL Burrow Integration <../integrations/index.html#hyperledger-burrow>`_.
```
Comment (stepanLav) [inline docs/source/build/index.rst:296]: It is required not only "go", but also the protobuf module, it is described here:
https://developers.google.com/protocol-buffers/docs/reference/go-generated

command with which you can install here:
"go install google.golang.org/protobuf/cmd/protoc-gen-go"

Comment (LiraLemur) [reply docs/source/build/index.rst]: I added the command and generally referenced the protobuf module. Please see :)

**Comment by lebdron** [comment]
As in previous pull request, can you fix your username and email in local git settings, so that you have commits that correspond to you github account? Thanks!

**Comment by LiraLemur** [comment]
> As in previous pull request, can you fix your username and email in local git settings, so that you have commits that correspond to you github account? Thanks!

@lebdron  I might try to find a way of doing that

**Review by stepanLav — APPROVED** [review]

**Comment by lebdron** [comment]
@LiraLemur there is this issue with just running `go install` at the moment https://stackoverflow.com/questions/62190610/cannot-find-package-google-golang-org-protobuf-cmd-protoc-gen-go . @stepanLav suggests to provide link to protobuf documentation instead

**Review by lebdron — APPROVED** [review]

**Pull request merged**